tests/test_html2docx.py

```diff
@@ -37,8 +38,10 @@ def test_html2docx(html_path, spec_path):
     for p, p_spec in zip(doc.paragraphs, spec):
         assert p.text == p_spec["text"]
         assert p.style.name == p_spec.get("style", "Normal")
-        if p.alignment:
+        if p_spec.get("alignment"):
```

Comment:
This won’t catch a paragraph with a non-default alignment when the spec does not specify. I think paragraphs can be assumed to have the no alignment unless otherwise specified in the spec.

I am suggesting the following mutation:
```diff
diff --git a/tests/test_html2docx.py b/tests/test_html2docx.py
index 772fd60..27c475e 100644
--- a/tests/test_html2docx.py
+++ b/tests/test_html2docx.py
@@ -40,8 +40,12 @@ def test_html2docx(html_path, spec_path):
         assert p.style.name == p_spec.get("style", "Normal")
         if p_spec.get("alignment"):
             assert int(p.alignment) == p_spec["alignment"]
+        else:
+            assert p.alignment is None
         if p_spec.get("left_indent"):
             assert p.paragraph_format.left_indent == Pt(p_spec["left_indent"])
+        else:
+            assert p.paragraph_format.left_indent is None
 
         runs_spec = p_spec["runs"]
         assert len(p.runs) == len(runs_spec)
```